backend/question_service2/README.md

```diff
@@ -0,0 +1,90 @@
+# Question Service
+
+This service manages Questions for PeerPrep. CRUD operations are supported. Questions are described by their Question Number (Primary Key), Title, Description, Category and Complexity.
+
+HTTP Status 200 means the operation completed successfully. Or else, other HTTP Statuses mean the operation faced some error.
+
+## How to start?
+
+Step 1. Install docker.
+
```

Comment:
Perhaps for this, and the other README for user service, can consider adding an additional step to say run docker. 

Because when I first tried these step, I followed the steps holistically only to realised my Docker instance is not running. 

I believe this will make the steps clearer.

Reply:
Done!